Dane/Intro/MainMenu/OUTRO/outro2/OUTRO_2.cnv

```diff
@@ -38,106 +48,358 @@ STATEK:PRELOAD=TRUE
 OUTRO3:PRELOAD=FALSE
 OUTRO3:RELEASE=FALSE
-OUTRO3:ONFINISHED={CR_OGRYZEK1^PLAY();}
-
-OBJECT=CR_OGRYZEK1
-CR_OGRYZEK1:TYPE=SOUND
-CR_OGRYZEK1:FILENAME=CR_OGRYZEK1.WAV
-CR_OGRYZEK1:PRELOAD=FALSE
-CR_OGRYZEK1:RELEASE=FALSE
-CR_OGRYZEK1:ONFINISHED={CR_BOSMAN1^PLAY();}
+OUTRO3:ONINIT=WEJW1
+OUTRO3:ONFINISHED=WEJW2
 
+#
+# CR_BOSMAN1
+#
 OBJECT=CR_BOSMAN1
 CR_BOSMAN1:TYPE=SOUND
 CR_BOSMAN1:FILENAME=CR_BOSMAN1.WAV
 CR_BOSMAN1:PRELOAD=FALSE
 CR_BOSMAN1:RELEASE=FALSE
-CR_BOSMAN1:ONFINISHED={CR_OGRYZEK2^PLAY();}
-
-OBJECT=CR_OGRYZEK2
-CR_OGRYZEK2:TYPE=SOUND
-CR_OGRYZEK2:FILENAME=CR_OGRYZEK2.WAV
-CR_OGRYZEK2:PRELOAD=FALSE
-CR_OGRYZEK2:RELEASE=FALSE
-CR_OGRYZEK2:ONSTARTED={CR_GRUBY1^PLAY();}
+CR_BOSMAN1:ONFINISHED=WEJW4
 
+#
+# CR_GRUBY1
+#
 OBJECT=CR_GRUBY1
 CR_GRUBY1:TYPE=SOUND
 CR_GRUBY1:FILENAME=CR_GRUBY1.WAV
 CR_GRUBY1:PRELOAD=FALSE
 CR_GRUBY1:RELEASE=FALSE
-CR_GRUBY1:ONSTARTED={CR_OGRYZEK3^PLAY();}
+CR_GRUBY1:ONFINISHED=WEJW6
+
+#
+# CR_MAJTEK1
+#
+OBJECT=CR_MAJTEK1
+CR_MAJTEK1:TYPE=SOUND
+CR_MAJTEK1:FILENAME=CR_MAJTEK1.WAV
+CR_MAJTEK1:PRELOAD=FALSE
+CR_MAJTEK1:RELEASE=FALSE
+CR_MAJTEK1:ONFINISHED=WEJW10
 
+#
+# CR_OGRYZEK1
+#
+OBJECT=CR_OGRYZEK1
+CR_OGRYZEK1:TYPE=SOUND
+CR_OGRYZEK1:FILENAME=CR_OGRYZEK1.WAV
+CR_OGRYZEK1:PRELOAD=FALSE
+CR_OGRYZEK1:RELEASE=FALSE
+CR_OGRYZEK1:ONFINISHED=WEJW3
+
+#
+# CR_OGRYZEK2
+#
+OBJECT=CR_OGRYZEK2
+CR_OGRYZEK2:TYPE=SOUND
+CR_OGRYZEK2:FILENAME=CR_OGRYZEK2.WAV
+CR_OGRYZEK2:PRELOAD=FALSE
+CR_OGRYZEK2:RELEASE=FALSE
+CR_OGRYZEK2:ONFINISHED=WEJW5
+
+#
+# CR_OGRYZEK3
+#
 OBJECT=CR_OGRYZEK3
 CR_OGRYZEK3:TYPE=SOUND
 CR_OGRYZEK3:FILENAME=CR_OGRYZEK3.WAV
 CR_OGRYZEK3:PRELOAD=FALSE
 CR_OGRYZEK3:RELEASE=FALSE
-CR_OGRYZEK3:ONFINISHED={CR_SOKOLEOKO1^PLAY();}
-
-OBJECT=CR_SOKOLEOKO1
-CR_SOKOLEOKO1:TYPE=SOUND
-CR_SOKOLEOKO1:FILENAME=CR_SOKOLEOKO1.WAV
-CR_SOKOLEOKO1:PRELOAD=FALSE
-CR_SOKOLEOKO1:RELEASE=FALSE
-CR_SOKOLEOKO1:ONSTARTED={CR_OGRYZEK4^PLAY();}
+CR_OGRYZEK3:ONFINISHED=WEJW7
 
+#
+# CR_OGRYZEK4
+#
 OBJECT=CR_OGRYZEK4
 CR_OGRYZEK4:TYPE=SOUND
 CR_OGRYZEK4:FILENAME=CR_OGRYZEK4.WAV
 CR_OGRYZEK4:PRELOAD=FALSE
 CR_OGRYZEK4:RELEASE=FALSE
-CR_OGRYZEK4:ONFINISHED={CR_MAJTEK1^PLAY();}
-
-OBJECT=CR_MAJTEK1
-CR_MAJTEK1:TYPE=SOUND
-CR_MAJTEK1:FILENAME=CR_MAJTEK1.WAV
-CR_MAJTEK1:PRELOAD=FALSE
-CR_MAJTEK1:RELEASE=FALSE
-CR_MAJTEK1:ONFINISHED={CR_OGRYZEK5^PLAY();}
+CR_OGRYZEK4:ONFINISHED=WEJW9
 
+#
+# CR_OGRYZEK5
+#
 OBJECT=CR_OGRYZEK5
 CR_OGRYZEK5:TYPE=SOUND
 CR_OGRYZEK5:FILENAME=CR_OGRYZEK5.WAV
 CR_OGRYZEK5:PRELOAD=FALSE
 CR_OGRYZEK5:RELEASE=FALSE
-CR_OGRYZEK5:ONSTARTED={OUTRO4^PLAY();}
+CR_OGRYZEK5:ONSTARTED=WEJWNAR2
+
+#
+# CR_SOKOLEOKO1
+#
+OBJECT=CR_SOKOLEOKO1
+CR_SOKOLEOKO1:TYPE=SOUND
+CR_SOKOLEOKO1:FILENAME=CR_SOKOLEOKO1.WAV
+CR_SOKOLEOKO1:PRELOAD=FALSE
+CR_SOKOLEOKO1:RELEASE=FALSE
+CR_SOKOLEOKO1:ONFINISHED=WEJW8
 
+#
+# OUTRO4
+#
 OBJECT=OUTRO4
 OUTRO4:TYPE=SOUND
 OUTRO4:FILENAME=OUTRO4.WAV
 OUTRO4:PRELOAD=FALSE
 OUTRO4:RELEASE=FALSE
-OUTRO4:ONFINISHED={RATUNEK^GOTO("OUTRO_3");}
+OUTRO4:ONFINISHED=WEJW11
```

Comment:
Same here and with all those subroutines down below. Having to jump to the bottom of script to be able to actually comprehend what happens next is tedious.